content/en/platform/corda/4.9/community/release-notes.md

```diff
@@ -17,6 +17,26 @@ tags:
 
 # Corda Community Edition 4.9 release notes
 
+## Corda Community Edition 4.9.7 release notes
+
+Corda Community Edition 4.9.7 is a patch release of Corda Community Edition which includes resolved issues.
+
+### Improvements
```

Comment:
Could you delete this and replace with Fixed Issues. Basically for a patch release we only fix issues, which these all out. (Will match other release notes as well).

Reply:
Done for both release notes.